WalletWasabi.Tests/Helpers/ArenaBuilder.cs

```diff
@@ -14,6 +14,8 @@ namespace WalletWasabi.Tests.Helpers;
 /// </summary>
 public class ArenaBuilder
 {
+	public static readonly ArenaBuilder Default = new();
```

Comment:
I'm a bit concerned about this because it is possible to modify this `ArenaBuilder` by setting a different `Period` or whatever.

Maybe we can instead do:

```suggestion
	public static ArenaBuilder Default => new();
```

to return always a new instance.